wordpress-mobile/WordPress-iOS
# My Site Dashboard: show ghost cards when loading for the first time

Part of #17873

Show a "ghost card" when the dashboard loads for the first time:

https://user-images.githubusercontent.com/7040243/156227242-fdf6dc09-7016-4854-a3ff-6a8a59dba622.mp4

**Notice:** the default loading indicator is still shown in the middle of the dashboard, which will be changed once we address showing dashboard updating feedback. Plus, failure and stale data will be handled in another task.

### To test

1. Remove the app from your device/simulator
2. Run the app
3. Enable the My Site Dashboard feature flag and restart the app
4. Tap "Home"
5. Check should see the ghost cards being displayed
6. Once the response from the API arrives, check that they're gone 

(Please test light/dark mode)

## Regression Notes
1. Potential unintended areas of impact
n/a

3. What I did to test those areas of impact (or what existing automated tests I relied on)
n/a

4. What automated tests I added (or what prevented me from doing so)
n/a

PR submission checklist:
